Makefile

```diff
@@ -3,6 +3,7 @@ PROJECT = landing
 COMMANDS = api
 CODECOV_TOKEN ?=
 DESTINATION ?= public
+DOCKERFILES = Dockerfile:landing Dockerfile.api:landing-api
```

Comment:
This is out of the scope of this PR, but I assume this variable should contain Dockerfile.slackin at some point in the future...